apps/shooting-clubs/translations/src/en/pages.json

```diff
@@ -10,7 +10,7 @@
   },
   "second-contact-name": {
     "header": "Who should we contact in {{values.club-secretary-name}}'s absence?",
```

Comment:
Yes pretty much, vijay wrote Jon O'Donneil.  I tested that too and that has the the same behaviour.